### PR TITLE
Fix graph not showing bug

### PR DESCRIPTION
**Pull request by daresh**
Fix statistics graph not working for PHP 7.

| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Products stats graph not showing bug fix related to PHP version.
| Type?             | bug fix
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes https://github.com/PrestaShop/PrestaShop/issues/13178
| How to test?      | Just go to Statistics - Product detail and check if the graph works
| Possible impacts? | Rather none.

### CODE REVIEW

File: statsproduct.php [hunk 1]
```diff
@@ -387,6 +387,7 @@ public function setOption($option, $layers = 1)
                 $this->_titles['main'][0] = $this->trans('Popularity', array(), 'Modules.Statsproduct.Admin');
                 $this->_titles['main'][1] = $this->trans('Sales', array(), 'Admin.Global');
                 $this->_titles['main'][2] = $this->trans('Visits (x100)', array(), 'Modules.Statsproduct.Admin');
+		$this->query = [];
```
Comment (kpodemski) [inline statsproduct.php:390]: It seems that indentation is wrong

**Review by kpodemski — CHANGES_REQUESTED** [review]

File: statsproduct.php (hunk 1 above)
Comment (daresh) [reply statsproduct.php:390]: Indeed, how to modify it now?

Comment (kpodemski) [reply statsproduct.php:390]:
```suggestion
		  $this->query = [];
```

Comment (kpodemski) [reply statsproduct.php]: Done ;)

**Review by kpodemski — APPROVED** [review]

**Review by kpodemski — APPROVED** [review]

**Comment by kpodemski** [comment]
Thanks @daresh 

Next time remember to use the `dev` branch 👍

**Review by Progi1984 — APPROVED** [review]

**Comment by daresh** [comment]
> Thanks @daresh
> 
> Next time remember to use the `dev` branch 👍

I'll try, but I'm not very fluent in GitHub, trying my best to help with bugs that I find.

File: statsproduct.php (hunk 1 above)
Comment (daresh) [reply statsproduct.php]: Thanks for help :)

**Comment by khouloudbelguith** [comment]
Hi @daresh,

It is ok :heavy_check_mark: 

**Without PR**
image

**With PR**
image


I checked with 177x, 178x and the develop branch.

Thanks!

**Comment by Progi1984** [comment]
Thanks @daresh & @khouloudbelguith

**Pull request merged**